### DISCUSSION TITLE
osuTK.Vector2 conversion

### DISCUSSION THREAD

**Discussion by Sword352** [ppy/osu-framework#5447] [Q&A] [answered]
Hi, just wanna tell that can we convert a number variable (int, float etc) into a osuTK.Vector2? Im making a rythm game and i dont know how to set a sprite's size because of that error saying that i cant convert the int into "osuTk.Vector2". Does i have to use a directive? If yes, what directive?
I'm french so the error is french, but i can translate : it says "Impossible to convert implicitly the int type into osuTK.Vector2"

![image](https://user-images.githubusercontent.com/99792114/194113738-022136f0-2910-403d-8a67-dd9c6e12abc8.png)

![image_2022-10-05_183253322](https://user-images.githubusercontent.com/99792114/194113572-578b927e-dbd8-4002-be4b-48769136705a.png)

![image_2022-10-05_182918092](https://user-images.githubusercontent.com/99792114/194112852-2aa6b415-0e0d-40c2-bae2-aa0947d4d08d.png)

**Comment by peppy** [comment, marked as answer]
You will need to use the correct type.

I'd recommend doing some tutorials on c# to understand types.

**Reply by Sword352** [reply]
Thanks!
